examples/quickstart-fastai/requirements.txt

```diff
@@ -1,3 +1,2 @@
-fastai~=2.7.12
-flwr~=1.4.0
-torch~=2.0.1
```

Comment:
@danielnugraha is torch not required anymore?

Reply:
Running `poetry install` will install torch automatically. It's already a fastai dependency I guess.

Reply:
@danielnugraha, any dependency we explicitly import in the code must also be a dependency in the requirements.txt as otherwise, removing one dependency might break modules that do not import that one.

Reply:
@danielnugraha, any dependency we explicitly import in the code must also be a dependency in the requirements.txt as otherwise, removing one dependency might break modules that do not import that one.

Reply:
Added torch as dependency.